trending_data/src/main/webapp/map_style.css

```diff
@@ -50,7 +57,7 @@ iframe {
 .yt-metadata-tag {
   color: white;
   font-weight: bold;
-  background-color: #c4302b;
+  background-color: #f00;
```

Comment:
Shouldn't this be 6 hex digits? What is `f00`?

Reply:
I think this might have something to do with the linter shortening the color code. What I wrote initially had a 6 digit code, but was shortened in this way: ff0000 -> f00.
 [See shorthand hex colors ](http://www.websiteoptimization.com/speed/tweak/hex/)

Reply:
Didn't know about this! Thanks :)